src/app.js

```diff
@@ -0,0 +1,29 @@
+require("dotenv").config();
+const express = require('express');
+const connectDB = require('./database');
+const errorHandler = require("./middlewares/errorHandler.middleware");
+const cartRouter = require("./routes/cart.route");
+const productRouter = require("./routes/products.route");
+const userRouter = require("./routes/user.route");
+const wishlistRouter = require("./routes/wishlist.route");
+const cors = require('cors');
+const app = express()
+const port = process.env.PORT || 3000;
+
+connectDB();
+
+app.use(cors());
+app.use(express.json())
+
+app.get('/', (req, res) => {
+  res.json({data: "Server is up and running"});
+});
+
+app.use('/cart', cartRouter);
+app.use('/products', productRouter);
+app.use('/users', userRouter);
+app.use('/wishlist', wishlistRouter);
```

Comment:
Instead of giving auth in separate files
You can add auth here only 
e.g: `app.use('/cart', auth, cartRouter)`
:) Then no need to import into individual files